rskj-core/src/test/java/co/rsk/peg/BridgeSupportTestUtil.java

```diff
@@ -122,6 +133,13 @@ public static BtcTransaction getReleaseFromPegoutsWFC(BridgeStorageProvider brid
         return pegoutEntry.getBtcTransaction();
     }
 
+    public static Block getBlockWithBlockNumber(long blockNumber) {
```

Comment:
```suggestion
    public static Block buildBlock(long blockNumber) {
```

Makes more sense to me.

Reply:
updated